osu.Game/Screens/Play/SkipOverlay.cs

```diff
@@ -87,30 +87,28 @@ private void load(OsuColour colours, GameplayClock clock)
             };
         }
 
-        private const double skip_required_cutoff = 3000;
         private const double fade_time = 300;
 
-        private double beginFadeTime => startTime - skip_required_cutoff - fade_time;
+        private double beginFadeTime => startTime - fade_time;
 
         protected override void LoadComplete()
         {
             base.LoadComplete();
-
-            if (startTime < skip_required_cutoff)
+            if (startTime + 2000 > 1000)
```

Comment:
This conditional feels backwards now. You are early returning the case we care about, not the early exit case.

Also, where did these values come from? And why does it make more sense to write this as `startTime + 2000 > 1000` instead of `startTime > -1000` or something like that?

Reply:
I'm gonna reverse it in that case. I wanted to use `> -1000` too, but I thought it would be more intuitive and neat since u can consider left part as literally `first object start time`, since `startTime` is 2 secs earlier. I'm gonna make both changes when I get some time.

Reply:
Where do you know the 2 seconds from? If there's another place this is used then please define it as a shared constant. Right now you have two random numbers in a statement with no explanation.

Reply:
I hope I fixed all the problems with the next commit.